src/store/indexeddb.ts

```diff
@@ -151,6 +151,13 @@ export class IndexedDBStore extends MemoryStore {
             });
     }
 
+    /*
+     * Close the database and destroy any associated workers
+     */
+    public destroy(): Promise<void> {
+        return this.backend.destroy();
+    }
```

Comment:
> Also also, any reason not to do this automatically in stopClient?

The `connect` method isn't called by `startClient` - and seemingly you might want to be able to start and stop your client independently from your stores, i.e stop your client when the network is down without tearing down your worker etc

Reply:
Re naming, it could be `close` `terminate` `cleanup` `stop` but we seem to use destroy generically here, at least in the react-sdk, where we even have an `IDestroyable` iface.

Happy to move it to IStore though